tasks/html-css-popup.md

```diff
@@ -0,0 +1,47 @@
+[![MIT Licensed][icon-mit]][license]
+[![Awesome][icon-awesome]][awesome]
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+[![Telegram][icon-chat]][chat]
+
+# HTML & CSS practice: Google-style Popup
+
+The goal of this task is to practice coding HTML & CSS from mockup. You are expected to deepen your knowledge of HTML & SCC and discover new previously unknown features.
+
+Requirements:
+- no JavaScipt
```

Comment:
`no JavaScript, only HTML/CSS use of preprocessors allowed.`
How you look on that?

Reply:
sure, good point